src/newbank/server/NewBank.java

```diff
@@ -42,7 +42,62 @@ public static NewBank getBank() {
         return bank;
     }
 
+    //TODO: not needed any more
```

Comment:
Remove the test Data as this now works with JSON.

Reply:
Done, all code involves using the test Data now removed.